extension/src/cli/cwd.ts

```diff
@@ -0,0 +1,11 @@
+import { realpathSync } from 'fs'
+
+export const getCaseSensitiveCwd = (path: string): string => {
+  try {
+    const cwd = realpathSync.native(path)
+    if (cwd) {
+      return cwd
+    }
+  } catch {}
+  return path
```

Comment:
[F] In this function I am trying to pre-empt everything that could go wrong when attempting to call `fs.realpathSync.native`.